### PR TITLE
Iarzuaga/sl 166/home carrousel

### PR DESCRIPTION
**Pull request by iarzuaga**

### CODE REVIEW

File: smallslive/artist_dashboard/views.py [hunk 1]
```diff
@@ -236,6 +236,7 @@ class MyPastEventsAJAXView(MyEventsAJAXView, MyPastEventsView):
     template_name = 'artist_dashboard/artist-dashboard-events.html'
     def get_context_data(self, **kwargs):
         context = super(MyPastEventsAJAXView, self).get_context_data(**kwargs)
+        context['is_past'] = True
```
Comment (rlopezcc) [inline smallslive/artist_dashboard/views.py]: Please remove this.

File: smallslive/templates/account/user_settings_new.html [hunk 2]
```diff
@@ -120,8 +130,11 @@
 				<div class="title5 account-settings-title">Billing</div>
 					<div class="account-settings-info">
 						<div class="text5">Credit card</div>
-						<div class="text9 text-grey">xxxx-xxxx-xxxx-xxxx-{{customer_detail.active_card.last4}}</div>
+						{% if not customer_detail.active_card %}
```
Comment (rlopezcc) [inline smallslive/templates/account/user_settings_new.html]: Please indent inside `if`

File: smallslive/templates/account/user_settings_new.html [hunk 3]
```diff
@@ -149,7 +163,17 @@
...
+		$infoId = $(this).data('id')
+		$infoContainer = $("#" + $infoId)[0]
+		$infoContainer.scrollIntoView()
+		$header = $("header")[0]
+		$header.scrollIntoView()
+	})
+
+=======
 	});
+>>>>>>> 84bf7a2bc0b1497d4ea0455d7c30b91a08fd32bf
```
Comment (rlopezcc) [inline smallslive/templates/account/user_settings_new.html]: I think this is a conflict.

File: smallslive/templates/artist_dashboard/artist-dashboard-events-info.html [hunk 4]
```diff
@@ -74,6 +74,7 @@
         </div>
     </div>
     </div>
+    {% if not event.is_future%}
```
Comment (rlopezcc) [inline smallslive/templates/artist_dashboard/artist-dashboard-events-info.html]: Please indent inside `if`.

File: smallslive/templates/artist_dashboard/artist-dashboard-events-info.html [hunk 5]
```diff
@@ -132,7 +145,11 @@
     </div>
     <div class="flex-column mobile-event-menu">
             <div id="mobile-metrics-button" class="flex-row mobile-event-opt">
+                {% if event.is_future%}    
+                <div class="white-border-button" disabled style="color: grey">Metrics</div>
```
Comment (rlopezcc) [inline smallslive/templates/artist_dashboard/artist-dashboard-events-info.html]: Please indent inside `if`.

File: smallslive/artist_dashboard/views.py (hunk 1 above)
Comment (iarzuaga) [reply smallslive/artist_dashboard/views.py]: Done

File: smallslive/templates/account/user_settings_new.html (hunk 2 above)
Comment (iarzuaga) [reply smallslive/templates/account/user_settings_new.html]: Done

File: smallslive/templates/artist_dashboard/artist-dashboard-events-info.html (hunk 4 above)
Comment (iarzuaga) [reply smallslive/templates/artist_dashboard/artist-dashboard-events-info.html]: Done

**Pull request merged**